### DISCUSSION TITLE
What's the difference between NFM and NF fonts?

### DISCUSSION THREAD

**Discussion by richardhttps** [ryanoasis/nerd-fonts#945] [General]
All of the fonts I've downloaded are monospaced so I don't get the difference.

JetBrainsMONO NF
JetBrainsMONO NFM

They look like duplicates except the name and font size.

**Comment by Finii** [comment]
These are the 'Windows Compatibles' names for

`NF` -> `Nerd Font`
`NFM` -> `Nerd Font Mono`

This means the two fonts are called (in normal, not Windows compat versions):
`JetBrainsMono Nerd Font` and
`JetBrainsMono Nerd Font Mono`.

(See #891 for the reasons why "Windows Compat" fonts have different names.)

The difference is in the added symbols. With `Nerd Font Mono` also the symbols are 'monospaced' (i.e. fit into the one 'cell'), while `Nerd Font` has bigger symbols (usually 1.3 to 1.8 cells wide). Although not strictly speaking monospaced they can be used in the usual monospaced contexts and just the symbols occupy more than one 'cell'. People often prefer these bigger symbols.

The `JetBrainsMono` part is the source font name.
`Nerd Font Mono` here Mono means 'single spaced', the word `Mono` is kind of a misnomer, but that has historic reasons.

### Example

Take this image from Issue 876

![image](https://user-images.githubusercontent.com/16012374/192545103-c47cc352-c5a5-4647-8a3b-2aec8fab1500.png)

The top image shows a `Nerd Font Mono` variant with smallish symbols, in the bottom image the symbols look more correct (maybe).
BUT they are bigger than one 'cell', this is a `Nerd Font` (non-Mono):

![image](https://user-images.githubusercontent.com/16012374/192545609-d9fe0503-1707-4377-99c1-81aac66ec7de.png)

_Edit: Fix typo, add link to 891_

**Reply by richardhttps** [reply]
That clears it up, thanks.
